pkg/evaluators/metadata/uma.go

```
@@ -138,7 +138,7 @@ func (pat *PAT) Get(rawurl string, ctx gocontext.Context, v interface{}) error {
 
 	otel.GetTextMapPropagator().Inject(ctx, otel_propagation.HeaderCarrier(req.Header))
 	// get the response
-	resp, err := httputil.NewClient().Do(req)
+	resp, err := httputil.NewClient(nil).Do(req)
```

golangci-lint (2.12.2)

[error] response body must be closed

(bodyclose)